common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -70,6 +70,12 @@
     ],
     "sqlState" : "42000"
   },
+  "AMBIGUOUS_RESOLVER_EXTENSION" : {
+    "message" : [
+      "The single-pass analyzer cannot process this query or command because the extension choice for <operator> is ambiguous: <extensions>."
+    ],
+    "sqlState" : "0A000"
```

Comment:
Let's change this sqlState since the error class `0A` belongs to unsupported features.

Reply:
Let's introduce new one, for instance `42705`. BTW, should users treat this error as their fault or internal Spark's compiler error?

Reply:
Mostly internal. But if it's a user-specified extension - this error will be thrown in case it's misconfigured.